python/grass/temporal/temporal_algebra.py

```diff
@@ -3325,8 +3325,50 @@ def p_expr_t_shift(self, t):
             elif len(t) == 7:
                 print(str(t[3]) + "* = tshift(", str(t[3]), ",", str(t[5]), ")")
 
-    # Handle errors.
+    def p_expr_time_const(self, t):
+        # Examples
+        # start_doy(A, -1)  # Get the start DOY from the preceding map
+        #                     of the time series as a numerical constant
+        #                     for the mapcalculator expression
+
+        """
+        expr : t_var LPAREN NAME COMMA INT RPAREN
+        """
+        if self.run:
+            # Check input stds.
+            maplist = self.check_stds(t[3])
+            resultlist = []
```

Comment:
While the names are consistent with the code around, there is not much reason to be consistent in this case and the resulting name is not a word, so an underscore would probably be nice. One day we will have to apply spell checker and less we have to fix or ignore, the better. Similarly for arith below in tests, although that's function names, not just local variables.

Reply:
Thanks. Much better. Resolved.